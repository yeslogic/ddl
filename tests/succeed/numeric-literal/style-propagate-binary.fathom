//~ test-normalisation = true

u8_add 0b1 2
