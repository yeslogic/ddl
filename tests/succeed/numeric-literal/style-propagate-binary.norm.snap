stdout = '''
0b11 : U8
'''
stderr = ''
